### PR TITLE
Add means to open specific tab with url

### PR DESCRIPTION
**Pull request by rainemak**
Developed on top of newer browser version and cherry-picked on top of next. However, changes are really minor when comparing esr45 and next branches.

### CODE REVIEW

**Comment by rainemak** [comment]
@jpetrell maybe you'd like to review?

**Comment by rainemak** [comment]
Rebased

File: src/core/declarativewebcontainer.h [hunk 1]
```diff
@@ -130,11 +132,13 @@ class DeclarativeWebContainer : public QWindow, public QQmlParserStatus, protect
     bool activatePage(const Tab& tab, bool force = false, int parentId = 0);
     int findParentTabId(int tabId) const;
 
-    Q_INVOKABLE void load(QString url = "", QString title = "", bool force = false);
+    Q_INVOKABLE void load(QString url = "", bool force = false);
```
Comment (mkenttala) [inline src/core/declarativewebcontainer.h]: Just more of a style thing but everywhere else strings have been converted to references but not this one

Comment (rainemak) [reply src/core/declarativewebcontainer.h]: Heck... I'll replace them all. :)

Comment (rainemak) [reply src/core/declarativewebcontainer.h]: All fixed now and changed amended to the style fix commit.

**Pull request merged**